src/components/Header/HeaderCart/CartList/CartList.jsx

```diff
@@ -0,0 +1,87 @@
+import './CartList.css';
+import '../HeaderCart.css';
+import emptyCartImg from '../../../../../public/img/empty-cart.png';
+
+export  default function CartList({ isOpenHeaderCartModal, headerCartProducts, setHeaderCartProducts }) {
+    return (
+        <>
+            {isOpenHeaderCartModal && (
+                <div className="cartListContainer">
+                    <div
+                        className="cartListRow"
+                        onClick={e => e.stopPropagation()}
+                    >
+                        <div className={`emptyCartImageRow ${headerCartProducts.length > 0 ? "disable" : ""}`}>
+                            <img
+                                className="emptyCartImage"
+                                src={emptyCartImg}
+                                alt="emptyCartImage"
+                            />
+                            <h2 className="cartInfo">Your Cart is Empty!</h2>
+                        </div>
+
+                        {headerCartProducts.map(product => (
+                            <ul className="cartListProductItem"
+                                key={product.id}
+                                id={product.id}
+                                onAnimationEnd={() => {
+                                    setHeaderCartProducts(
+                                        headerCartProducts.filter(a =>
```

Comment:
нормальну назву підбираєм в колбеку. Краще переназви на item 
```suggestion
                                        headerCartProducts.filter(item =>
```